app/org/sagebionetworks/bridge/services/SendMailViaAmazonService.java

```diff
@@ -87,29 +101,54 @@ public void sendConsentAgreement(User user, ConsentSignature consentSignature, S
             if (consentSignature.getImageData() != null) {
                 // Use pre-encoded MIME part since our image data is already base64 encoded.
                 sigPart = new PreencodedMimeBodyPart(HEADER_CONTENT_TRANSFER_ENCODING_VALUE);
-                sigPart.setContentID(HEADER_CONTENT_ID_VALUE);
-                sigPart.setHeader(HEADER_CONTENT_DISPOSITION, HEADER_CONTENT_DISPOSITION_VALUE);
+                sigPart.setContentID(HEADER_CONTENT_ID_CONSENT_VALUE);
+                sigPart.setHeader(HEADER_CONTENT_DISPOSITION, HEADER_CONTENT_DISPOSITION_CONSENT_VALUE);
                 sigPart.setContent(consentSignature.getImageData(), consentSignature.getImageMimeType());
             }
             
             // As recommended by Amazon, we send the emails separately.
-            sendEmailTo(study.getName(), user.getEmail(), bodyPart, sigPart);
+            String subject = String.format(CONSENT_EMAIL_SUBJECT, study.getName());
+            
+            sendEmailTo(subject, user.getEmail(), bodyPart, sigPart);
             Set<String> emailAddresses = commaListToSet(study.getConsentNotificationEmail());
             for (String email : emailAddresses) {
-                sendEmailTo(study.getName(), email, bodyPart, sigPart);
+                sendEmailTo(subject, email, bodyPart, sigPart);
             }
         } catch(IOException | MessagingException e) {
             throw new BridgeServiceException(e);
         }
     }
+    
+    @Override
+    public void sendStudyParticipantsRoster(Study study, List<StudyParticipant> participants) {
```

Comment:
We should create a unit test for this, with mocks so we can verify the emails sent are reasonable.